compliance/google/long_stopped_instances/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+v1.0
```

Comment:
needs Changelog header

Reply:
I have addressed, please review this.
thanks